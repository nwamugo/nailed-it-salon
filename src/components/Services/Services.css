.line-boundary {
  position: absolute;
  top: 43rem;
  width: 2.95rem;
  height: 0.18rem;
  background-color: #000;
  left: 57.56rem;
}

.services-container {
  position: absolute;
  top: 46rem;
  left: 14rem;
  height: 30.981875rem;
  width: 86.3125rem;
}

.primary-header {
  text-align: center;
  text-transform: uppercase;
  font-family: 'Josefin Sans', sans-serif;
  font-size: 2.8125rem;
  font-weight: 600;
  letter-spacing: .04em;
}