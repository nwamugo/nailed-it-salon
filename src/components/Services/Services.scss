.serving {
  width: 100%;
  padding: 0;
  margin: 0;
  margin-top: -13.5%;
}
.serving-board {
  width: 85%;
  margin: 0 auto;
  padding: 0;
  text-align: center;
}
.services-container-border-line {
  background-color: #000;
  width:3%; 
  margin: auto;
  height: 0.288rem;
  margin-bottom: 20px;

  @media (min-width: 320px) and (max-width: 480px) {
    width:7%; 
  }
  @media (min-width: 481px) and (max-width: 767px) {
    width:5%; 
  }
  @media (min-width: 768px) and (max-width: 1024px) {
    width:4%; 
  }
}
.services-container {
  margin: 7rem 0;
  align-items: center;
  display: flex;
  align-content: center;
  flex-direction: column;
}
.services-container .primary-header {
  text-transform: uppercase;
  font-family: "Josefin Sans", sans-serif;
  font-size: 4.5rem;
  font-weight: 600;
  letter-spacing: 0.04em;
}
.services-container .container-fluid {
  width: 140rem;
  margin-top: 15px;

  @media (min-width: 320px) and (max-width: 1024px) {
    width: 70rem; 
  }
}
