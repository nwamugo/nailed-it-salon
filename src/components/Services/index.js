import React from 'react';

import ServicesCard from '../ServicesCard';
import Pedicure from '../../shared/images/pedicure.png';
import NailsExtension from '../../shared/images/nailsextension.png';
import Manicure from '../../shared/images/manicure.png';
import Addons from '../../shared/images/add-ons.png';

import './Services.scss';

const Services = () => {
  return (
    <div className="serving" id="serving">
      <div className="serving-board">
        <hr className="services-container-border-line" />
        <div className="services-container">
          <h2 className="primary-header">Our Services</h2>
          <div className="container-fluid">
            <div className="row">
              <ServicesCard
                className="col"
                image={Manicure}
                cardTitle="Manicure"
              />
              <ServicesCard
                className="col"
                image={Pedicure}
                cardTitle="Pedicure"
              />
            </div>
            <div className="row">
              <ServicesCard
                className="col"
                image={NailsExtension}
                cardTitle="Nail Extensions"
              />
              <ServicesCard
                image={Addons}
                className="col"
                cardTitle="Add Ons"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Services;
