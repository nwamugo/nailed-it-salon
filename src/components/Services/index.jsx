import React from 'react';
import './Services.css';

const Services = () => {
  return (
    <div>
      <hr className="line-boundary" />

      <div className="services-container">
        <h2 className="primary-header">Our Services</h2>
      </div>
    </div>
  );
};

export default Services;
