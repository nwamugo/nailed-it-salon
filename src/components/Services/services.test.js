import React from 'react';
import { shallow } from 'enzyme';
import Services from './index';

describe('Services', () => {
  it('should render correctly', () => {
    const component = shallow(<Services />);

    expect(component).toMatchSnapshot();
  });
});
