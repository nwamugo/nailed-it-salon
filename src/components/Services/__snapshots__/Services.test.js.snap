// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services should not have regressed 1`] = `
<div
  className="serving"
  id="serving"
>
  <div
    className="serving-board"
  >
    <hr
      className="services-container-border-line"
    />
    <div
      className="services-container"
    >
      <h2
        className="primary-header"
      >
        header.services
      </h2>
      <div
        className="container-fluid"
      >
        <div
          className="row"
        >
          <ServicesCard
            cardTitle="services.manicure"
            className="col"
            image=""
          />
          <ServicesCard
            cardTitle="services.pedicure"
            className="col"
            image=""
          />
        </div>
        <div
          className="row"
        >
          <ServicesCard
            cardTitle="services.extensions"
            className="col"
            image=""
          />
          <ServicesCard
            cardTitle="services.addons"
            className="col"
            image=""
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
